googletest/include/gtest/gtest.h

```diff
@@ -1965,13 +1965,13 @@ class TestWithParam : public Test, public WithParamInterface<T> {
 // AssertionResult. For more information on how to use AssertionResult with
 // these macros see comments on that class.
 #define EXPECT_TRUE(condition) \
-  GTEST_TEST_BOOLEAN_(condition, #condition, false, true, \
+  GTEST_TEST_BOOLEAN_((condition), #condition, false, true, \
```

Comment:
Why did you add parenthesis?

Reply:
Wasnt meant to be pushed :D thanks

Reply:
Sure, you're welcome :) And carry on continuing my work :muscle: